server.go

```diff
@@ -166,6 +166,13 @@ type Server struct {
 	//   * ErrBrokenChunks
 	ErrorHandler func(ctx *RequestCtx, err error)
 
+	// ResponseErrorHandler is called in case of an error while writing the response
+	//
+	// Useful if you want to log details of write failures for compliance and
+	// business critical requests
+	// The handler doesn't need to modify the response
```

Comment:
Can you turn this into a big fat warning. I think if you do things to modify the response it will end up very badly!